test/sfs.py

```diff
@@ -117,12 +117,11 @@ def read(self, size=-1):
         return buffer.value
 
     def write(self, content_bytes):
-        content_len = len(content_bytes)
-        write_status = c_helper.nfs4_pwrite(self._file, content_bytes, self._pos, content_len)
-        if write_status != NFS4_OK:
-            if write_status == NFS4ERR_OPENMODE:
-                raise io.UnsupportedOperation("not writable") 
+        bytes_written = c_helper.nfs4_pwrite(self._file, content_bytes, self._pos, len(content_bytes))
+        if bytes_written < 0:
+            # if write_status == NFS4ERR_OPENMODE:
```

Comment:
I still need to dig out the error_num from file->nfs4->err_num

Reply:
Ok, 'looks like we're getting close.